fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeStmtAnalyzer.java

```diff
@@ -36,8 +38,8 @@ public void analyze(StatementBase statement, ConnectContext session) {
         /**
          * analyse user identity + check if user exists in UserPrivTable
          */
-        private void analyseUserAndCheckExist(UserIdentity userIdent, ConnectContext session) {
-            // validate user
+        private void analyseUserAndCheckExist(UserIdentity userIdent, ConnectContext session, boolean checkExist) {
```

Comment:
It would be weird to have an argument called `checkExists` in a function called `analyseUserAndCheckExists`. Newer developers would be confused about whether this function checks the existence of the user or not. In my own opinion, the name `analyseUser` would make more sense for this function.

Reply:
done